fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -19,14 +19,29 @@ def self.run(params)
           command_prefix,
           'agvtool',
           'what-marketing-version',
-          '-terse1'
```

Comment:
Is this change intensional? Local tests show that this changes the output of the command significantly.

```
asfalcone /internal-examples/SingleView $ agvtool what-marketing-version -terse1
1.0
asfalcone /internal-examples/SingleView $ agvtool what-marketing-version -terse
"SingleView.xcodeproj/../SingleView/Info.plist"=1.0
"SingleView.xcodeproj/../SingleViewTests/Info.plist"=1.0
"SingleView.xcodeproj/../SingleViewUITests/Info.plist"=1.0
```